pages/index.tsx

```diff
@@ -18,10 +18,10 @@ const Home: NextPage = () =>
                 <li>You must complete the test in one sitting</li>
                 <li>When you are ready and sure you won’t be disturbed, please click “Start”</li>
             </ul>
-            <Link passHref={true} href={"/testpage"} as={testToRender.title}>
-                <a className={scss.buttonYellow}>Start</a>
+            <Link href={"/testpage"} as={testToRender.title}>
+                <p className={scss.buttonYellow}>Start</p>
```

Comment:
This one seems like it should definitely be a link.

Reply:
Axe was grumbling at me because the href is in Link instead of the a, saying the link had no target. Tried putting href in a too but it messes up the dynamic url and redirects to a 404 page

Reply:
on staging it looks totally fine - the `Link` element shouldn't appear in the final HTML - its just a tiny wrapper around an `a` tag anyway.

Perhaps works differently if you run in dev mode?

Take a look here though - https://testswitch-candidate-staging.herokuapp.com/
HTML is just a normal `a` with a normal `href`

Reply:
Ah, done